File: BackEnd/ProjectManagerSystem/DataModelLibrary/Database/AtomEmployeePipelineSaler/DbAtomEmployeePipelineSalerStatusEnum.cs
namespace DataModelLibrary.Database.AtomEmployeePipelineSaler;

/// <summary>資料庫-原子-員工商機業務-狀態</summary>
public enum DbAtomEmployeePipelineSalerStatusEnum : short
{
    /// <summary>未定義</summary>
    Undefined = 0,

    /// <summary>尚未回覆</summary>
    Pending = 1,

    /// <summary>接受</summary>
    Accepted = 2,

    /// <summary>拒絕</summary>
    Rejected = 3,

    /// <summary>轉指派業務</summary>
    Reassigned = 4,
}

File: FrontEnd/manager-back-site/src/constants/DbAtomEmployeePipelineSalerStatusEnum.ts
/** 資料庫-原子-員工商機業務-狀態 */
export enum DbAtomEmployeePipelineSalerStatusEnum {
  /** 未定義 */
  Undefined = 0,
  /** 尚未回覆  */
  Pending = 1,
  /** 接受  */
  Accepted = 2,
  /** 拒絕  */
  Rejected = 3,
  /** 轉指派業務  */
  Reassigned = 4,
}
